api/nakadi-event-bus-api.yaml

```diff
@@ -1270,3 +1281,43 @@ definitions:
         description: Documentation for the enrichment.
     required:
       - name
+
+
+  BatchItemResponse:
+    description: |
+      A status corresponding to one individual Event's publishing attempt.
+    properties:
+      eid:
+        type: string
+        format: uuid
+        description: |
+          eid of the corresponding item.
+      publishing_status:
```

Comment:
- about yes/no: failed/submitted is the same; and there we had agreed on not being boolean. 
- about the step: is the same, but without failed. failed is no step, but an outcome. the names used are just more in line with what the api already specifies. the semantic is the same.